provider/pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -290,6 +292,10 @@ interface BaseChartOpts {
      * The optional namespace to install chart resources into.
      */
     namespace?: pulumi.Input<string>;
+    /**
+     * The optional flag to determine if CRDs from the chart will be installed
```

Comment:
Can we use the same language for all of the SDKs?

Reply:
Yes will get this fixed up